presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionKeywordSuite.scala

```diff
@@ -467,26 +464,16 @@ class CompletionKeywordSuite extends BaseCompletionSuite:
          |  def hello(u@@)
          |}""".stripMargin,
       """|using (commit: '')
-         |unsafe - scala.caps (commit: '')
-         |unsafeNulls - scala.runtime.stdLibPatches.language (commit: '')
-         |unused - scala.annotation (commit: '')
-         |unshared - scala.annotation.internal (commit: '')
          |""".stripMargin,
       includeCommitCharacter = true,
-      topLines = Some(5)
     )
 
   @Test def `not-using` =
     check(
       """|object A{
          |  def hello(a: String, u@@)
          |}""".stripMargin,
-      """|unsafe - scala.caps
-         |unsafeNulls - scala.runtime.stdLibPatches.language
-         |unused - scala.annotation
-         |unshared - scala.annotation.internal
-         |unspecialized - scala.annotation""".stripMargin,
-      topLines = Some(5)
+      "",
```

Comment:
why did this change?

Reply:
these are illegal completions at this position, it expects the name. On the other hand you can add annotations in this position. I'll add tests that ensure that they are present if we start a query with @